### ISSUE TITLE
TypeError: Constructor Map requires 'new' 

### ISSUE THREAD

**Issue by tanykim** [mbostock/internmap#3]
After updating d3-array 2.12.1 version, I came across this error `TypeError: Constructor Map requires 'new' ` while using `rollup` or `rollups`. I see the error stack: rollup > nest > regroup > InternMap#constructor).

 I had no errors with the previous version(change is [here](https://github.com/mbostock/internmap/commit/6d479c8ff2ccd6404d477003d30ce3a132253d2c#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556)). 

I'm working on top of Flow-typed repo, if that's relevant.

**Comment by mbostock** [comment]
I’m not able to reproduce, but I’d be happy to investigate if you provide a reproduction.

https://observablehq.com/@d3/d3-group

**Comment by bumbeishvili** [comment]
I also saw this error in one of my clients' projects. He was using Typescript. I could not reproduce it locally. 

This is an error screenshot on his side: 

![image001](https://user-images.githubusercontent.com/6873202/132465067-f66f5c4a-122c-49d8-ab4a-9efe711ac0cd.png)

This is a relevant line in the d3-org-chart codebase

https://github.com/bumbeishvili/org-chart/blob/98850af142167ad958b25eb09e78998b1e4b885b/src/d3-org-chart.js#L561

I ended up replacing the d3.rollup function with custom solution
